### PR TITLE
Dependency map

### PR DESCRIPTION
**Pull request by timbl**
Add a map of internal dependencies and the tool to make it.

### CODE REVIEW

File: src/folders.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ import * as debug from './debug'
 const UI = {
   icons: require('./iconBase'),
   ns: require('./ns'),
-  pad: require('./'),
```
Comment (megoth) [inline src/folders.js:10]: Is this change necessary in this PR?

File: src/matrix.js [hunk 2]
```diff
@@ -23,7 +23,6 @@
 var UI = {
   icons: require('./iconBase'),
   ns: require('./ns'),
-  pad: require('./'),
```
Comment (megoth) [inline src/matrix.js:26]: Is this change necessary in this PR?

File: src/messageArea.js [hunk 3]
```diff
@@ -4,7 +4,6 @@ var UI = {
   authn: require('./authn/authn'),
   icons: require('./iconBase'),
   ns: require('./ns'),
-  pad: require('./'),
```
Comment (megoth) [inline src/messageArea.js:7]: Is this change necessary in this PR?

File: src/folders.js (hunk 1 above)
Comment (timbl) [reply src/folders.js:10]: I removed 3 unnecessary dependencies at the same time.  Can we let that go through in the same PR?

File: src/matrix.js (hunk 2 above)
Comment (timbl) [reply src/matrix.js:26]: Not strictly but it is related and life is too short to separate them

File: src/messageArea.js (hunk 3 above)
Comment (timbl) [reply src/messageArea.js:7]: Not strictly but it is related and life is too short to separate them

**Review by michielbdejong — APPROVED** [review]
